src/components/image-contentBlock/image-contentBlock.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import styled from 'styled-components';
+import { down } from '../breakpoint/breakpoint';
+import { theme } from '../layout/theme';
+import { TextStyles } from '../typography/typography-v2';
+
+const BlockWrapper = styled.div<{
+  textAlign: 'right' | 'left' | 'bottom';
+}>`
+  display: inline-flex;
+  flex-direction: ${({ textAlign }) =>
+    textAlign === 'right'
+      ? 'row'
+      : textAlign === 'left'
+      ? 'row-reverse'
+      : 'column'};
+
+  width: 100%;
+
+  ${down('sm')} {
+    flex-direction: column;
+  }
+`;
+
+const ImageWrapper = styled.div`
+  display: inline;
+  position relative;
+`;
+
+const AttributionContainer = styled.div`
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  background: rgba(0, 0, 0, 0.2);
+  display: block;
+
+  padding: 6px 8px;
+  margin-bottom: 5px;
+
+  font-size: 10px;
+  line-height: 110%;
+  color: #ffffff;
+`;
+
+const StyledLink = styled.a`
+  color: #ffffff;
+`;
+
+const TextWrapper = styled.div<{
+  textAlign: 'right' | 'left' | 'bottom';
+}>`
+  ${({ textAlign }) =>
+    textAlign === 'right'
+      ? 'margin-left: 24px;'
+      : textAlign === 'left'
+      ? 'margin-right: 24px;'
+      : 'margin-top: 12px;'}
+
+  ${({ textAlign }) =>
+    textAlign === 'bottom' ? TextStyles.textXS : TextStyles.textS}
+
+  ${({ textAlign }) =>
+    textAlign === 'bottom' ? 'width 0; min-width:100%;' : ''}
+  ${({ textAlign }) => (textAlign !== 'bottom' ? 'max-width: 228px;' : '')}
+
+  ${down('sm')} {
+    ${TextStyles.textXS}
+
+    margin: 12px 0;
+    width 0;
+    min-width: 100%;
+    max-width: 100;
+  }
+
+  ${theme.palette.text.default};
+`;
+
+interface ImageContentBlockProps {
+  children: React.ReactNode;
+  textAlign: 'right' | 'left' | 'bottom';
+  description: string;
+  attribution: {
+    source: string;
+    creator: string;
+  };
+}
+
+const ImageContentBlock = ({
+  children,
+  textAlign,
+  description,
+  attribution,
+}: ImageContentBlockProps) => {
+  return (
+    <BlockWrapper textAlign={textAlign}>
+      <ImageWrapper>
+        {children}
+        {attribution && (
+          <AttributionContainer>
+            Photo by{' '}
```

Comment:
This is not covered by the translation system. Have you though about it ?

Reply:
I did not thought about it (probably since in Figma there is only "Photo by") and I thought that for this creator tag the english should then be fine (since the rest of the design is in german)...
Shall I support translation ? And who do I do this?

Reply:
Don't trust the design too much 🤓 Think about the final result and imaging being a Germany visitor. Would you like to read random english words while the page is German? Please fix.

Please ask @hofmann-felix for support as he used it before too. We use https://react.i18next.com/ so please read the documentation in addition a little bit to know the foundation as this will hunt you in many places anyway 👹

Reply:
Okay, yes, I implemented localisation

Reply:
thanks 🙌